sktime/transformations/series/temporian.py

```diff
@@ -0,0 +1,128 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""Implementation of TemporianTransformer. Contributed by the Temporian team."""
+
+__author__ = ["ianspektor"]  # TODO: add other Temporian devs who contributed to this
+
+from sktime.transformations.base import BaseTransformer
+
+
+# TODO: add usage example to docstring
+# see https://www.sktime.net/en/latest/developer_guide/dependencies.html#dependencies
+# for how to handle soft deps (temporian) there (need to not run doc tests on it)
+class TemporianTransformer(BaseTransformer):
+    """Applies a Temporian function to the input time series.
+
+    This transformer applies a [Temporian](https://temporian.readthedocs.io/en/stable/)
```

Comment:
this is ReStructuredText, not markdown, so this will not render correctly. Markdown has a peculiar link syntax with apostrophes, tag symbols, underscores, which I can't really describe here because this is markdown...

so I'm just adding a link:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
(in numpydoc style, the link should be in the "References" section, and referred to as `[1]_` or similar)